connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -328,29 +330,37 @@ export async function syncZendeskArticleBatchActivity({
     `[Zendesk] Processing ${articles.length} articles in batch`
   );
 
-  const sections =
-    await zendeskApiClient.helpcenter.sections.listByCategory(categoryId);
-  const { result: users } = await zendeskApiClient.users.showMany(
-    articles.map((article) => article.author_id)
-  );
+  try {
+    const sections =
+      await zendeskApiClient.helpcenter.sections.listByCategory(categoryId);
+    const { result: users } = await zendeskApiClient.users.showMany(
+      articles.map((article) => article.author_id)
+    );
 
-  await concurrentExecutor(
-    articles,
-    (article) =>
-      syncArticle({
-        connectorId,
-        category,
-        article,
-        section:
-          sections.find((section) => section.id === article.section_id) || null,
-        user: users.find((user) => user.id === article.author_id) || null,
-        dataSourceConfig,
-        currentSyncDateMs,
-        loggerArgs,
-        forceResync,
-      }),
-    { concurrency: 10 }
-  );
+    await concurrentExecutor(
+      articles,
+      (article) =>
+        syncArticle({
+          connectorId,
+          category,
+          article,
+          section:
+            sections.find((section) => section.id === article.section_id) ||
+            null,
+          user: users.find((user) => user.id === article.author_id) || null,
+          dataSourceConfig,
+          currentSyncDateMs,
+          loggerArgs,
+          forceResync,
+        }),
+      { concurrency: 10 }
+    );
+  } catch (e) {
+    if (isNodeZendeskForbiddenError(e)) {
```

Comment:
IMO you only want to wrap this in the tryCatch:
```
const sections =
      await zendeskApiClient.helpcenter.sections.listByCategory(categoryId);
    const { result: users } = await zendeskApiClient.users.showMany(
      articles.map((article) => article.author_id)
    );
```

the stuff happening in concurrentExecutor does not make any zendesk API calls, so wrapping those in try/catch just adds confusion

Reply:
true, I'll do that